extensions/adobe/experience/adcloud/experienceevent-all.schema.json

```diff
@@ -35,6 +35,11 @@
           "title": "Ad Asset Details",
           "$ref": "https://ns.adobe.com/experience/adcloud/advertisement",
           "description": "Digital advertisement details"
+        },
+        "https://ns.adobe.com/experience/adcloud/stitchIdChosen": {
+          "title": "Stitch ID Chosen Type",
+          "type": "string",
+          "description": "ID from the Ad Servers through AdCloud STATS to track click-through conversion on browsers that clock third party cookies"
```

Comment:
What does "clock third party cookies" mean?

Reply:
@cdegroot-adobe : ah, 🤦‍♂️ That's a typo. Should have been block. Will fix it in my next commit